api/views.py

```diff
@@ -49,3 +49,22 @@ def api_login():
         return jsonify(user.to_dict())
     else:
         return jsonify({'success': False}), 401
+
+
+@app.route('/api/position/', methods=['post'])
+def api_position():
```

Comment:
Mhh… we can't just send the point without login the user or anyone could spoof. #security-issue

We need to create an authentication token and send it in the headers (`Authentication` header).

Reply:
The best way would be to authenticate with Oauth2, but it may be overkill in our case.
Maybe we could just generate an temporary token when the user logs in from the mobile app and send the token when communicating with the API.
It's a weak authentication process, but at least it's one.

Reply:
Eventually we could just use the id for now for the sake of easiness, open a security issue and treat it in a second time.